fix/chat: Label the button that runs shell commands "Execute" instead of "Execute in Terminal"

dominiccooney opened the pull request:
Also move the ... overflow menu to be the last menu item on the left.

## Test plan

Tested manually: Ask Cody to generate a 10 line shell script, and verify that the button is labelled "Execute."

Check that the Copy and Execute icons are aligned, both in cases where there are filenames/Guardrails, and where there is no metadata.

![Screenshot 2025-03-28 at 16 55 11](https://github.com/user-attachments/assets/171ca72a-2bf6-4fda-98be-7ec7da0d7235)

abeatrix reviewed (approved):
is it expected to have the trailing menu in between Copy and Run?  Might make more sense to put [Run, Copy, ...] or [Copy, Run, ...]

jdorfman reviewed (commented):
The icon needs adjusting

https://www.loom.com/share/326f48eb5ef541e79c7eeaa9d86bc756

dominiccooney commented:
I have aligned the icons, it was a good opportunity to simplify the DOM here. Thank you for the attention to detail @jdorfman .

I have moved the ... overflow menu to the right of the left set of action buttons, thanks @abeatrix , that was a good call out.

I changed the name from "Run" to "Execute." I checked v1.80, it used to be labeled "Execute."

jdorfman commented:
@dominiccooney Execute is def better. Good call.
